### DISCUSSION TITLE
posts/tombkeeper-sec-learning/

### DISCUSSION THREAD

**Discussion by giscus** [brant-ruan/fernweh-comments#21] [Announcements]
# posts/tombkeeper-sec-learning/

tombkeeper关于信息安全学习的分享

https://blog.wohin.me/posts/tombkeeper-sec-learning/

**Comment by CalbeeMing0530** [comment]
赞博男，有些文章有时代背景，比如“信息安全的未来如何？”，如果可以加上发表时间就更好啦

**Reply by brant-ruan** [reply]
想着加上时间有点繁杂，就没加。而且根据内容其实也能基本上推断出来特别老的内容，比如你说的这个是2006年😄

**Comment by MC-box** [comment]
好厉害，我真的就是缺学技术的方式，学的又慢又不好，上大学后也是一直在找方法

**Reply by MC-box** [reply]
我跟大哥还是有方向上相似的，好想交流一下:sob:

**Reply by brant-ruan** [reply]
加油，另外可以通过邮箱联系我～
